config/boards/orangepi-r1plus.conf

```diff
@@ -0,0 +1,11 @@
+# Rockchip RK3328 quad core 1GB 2xGBE USB2 SPI
+BOARD_NAME="Orange Pi R1 Plus"
+BOARDFAMILY="rockchip64"
+BOOTCONFIG="orangepi_r1_plus_rk3328_defconfig"
+KERNEL_TARGET="current"
+DEFAULT_CONSOLE="serial"
+MODULES="g_serial"
+MODULES_BLACKLIST="rockchipdrm analogix_dp dw_mipi_dsi dw_hdmi gpu_sched lima hantro_vpu"
+SERIALCON="ttyS2:1500000,ttyGS0"
+BUILD_DESKTOP="no"
+BOOT_FDT_FILE="rockchip/rk3328-orangepi-r1-plus.dtb"
```

Comment:
BOOT_FDT_FILE is not needed since we are using dedicated boot loader config where default DT is defined. https://github.com/armbian/build/pull/2755/files#diff-369eba59eb8e91afb2b0a4b235855de1a011669550abe2d5d081d4ad83845a3eR113